### ISSUE TITLE
Error building Nef_3 with NOT EPECK

### ISSUE THREAD

**Issue by haelue** [CGAL/cgal#4840]
Today I switch Kernel Epeck to Epick, it comes out strange building ERRORS (Ubuntu 16.04, GCC 5.4.0). I use cgal for half a year now, and suggest it to be a bug.

Errors in console as belows:

```
[build] In file included from /usr/local/include/CGAL/Nef_3/SNC_k3_tree_traits.h:19:0,
[build]                  from /usr/local/include/CGAL/Nef_3/SNC_point_locator.h:21,
[build]                  from /usr/local/include/CGAL/Nef_3/SNC_external_structure.h:25,
[build]                  from /usr/local/include/CGAL/Nef_3/SNC_constructor.h:32,
[build]                  from /usr/local/include/CGAL/Nef_3/SNC_decorator.h:29,
[build]                  from /usr/local/include/CGAL/Nef_polyhedron_3.h:28,
[build]                  from /root/Documents/project/geometricserver/./include/cgal/cgal_common.h:17,
[build]                  from /root/Documents/project/geometricserver/function/intersectinfo/collidearea.h:15,
[build]                  from /root/Documents/project/geometricserver/function/intersectinfo/collidearea.cpp:1:
[build] /usr/local/include/CGAL/Nef_3/Bounding_box_3.h: In instantiation of ‘void CGAL::Bounding_box_3<CGAL::Boolean_tag<true>, Kernel>::extend(const Point_3&) [with Kernel = CGAL::Simple_cartesian<double>; CGAL::Bounding_box_3<CGAL::Boolean_tag<true>, Kernel>::Point_3 = CGAL::Point_3<CGAL::Simple_cartesian<double> >]’:
[build] /usr/local/include/CGAL/Nef_3/SNC_k3_tree_traits.h:215:4:   required from ‘CGAL::Objects_bbox<SNC_decorator>::Bounding_box_3 CGAL::Objects_bbox<SNC_decorator>::operator()(const Object_list&) const [with SNC_decorator = CGAL::SNC_decorator<CGAL::SNC_structure<CGAL::Simple_cartesian<double>, CGAL::SNC_indexed_items, bool> >; CGAL::Objects_bbox<SNC_decorator>::Bounding_box_3 = CGAL::Bounding_box_3<CGAL::Boolean_tag<true>, CGAL::Simple_cartesian<double> >; typename SNC_decorator::Kernel = CGAL::Simple_cartesian<double>; CGAL::Objects_bbox<SNC_decorator>::Object_list = std::vector<CGAL::Object>; typename SNC_decorator::SNC_structure::Object_handle = CGAL::Object]’
[build] /usr/local/include/CGAL/Nef_3/K3_tree.h:854:29:   required from ‘CGAL::K3_tree<Traits>::K3_tree(CGAL::K3_tree<Traits>::Object_list&, CGAL::K3_tree<Traits>::Object_iterator&) [with Traits = CGAL::SNC_k3_tree_traits<CGAL::SNC_decorator<CGAL::SNC_structure<CGAL::Simple_cartesian<double>, CGAL::SNC_indexed_items, bool> > >; CGAL::K3_tree<Traits>::Object_list = std::vector<CGAL::Object>; typename Traits::Object_handle = CGAL::Object; CGAL::K3_tree<Traits>::Object_iterator = __gnu_cxx::__normal_iterator<CGAL::Object*, std::vector<CGAL::Object> >]’
[build] /usr/local/include/CGAL/Nef_3/SNC_point_locator.h:399:24:   required from ‘void CGAL::SNC_point_locator_by_spatial_subdivision<SNC_decorator>::initialize(CGAL::SNC_point_locator_by_spatial_subdivision<SNC_decorator>::SNC_structure*) [with SNC_decorator = CGAL::SNC_decorator<CGAL::SNC_structure<CGAL::Simple_cartesian<double>, CGAL::SNC_indexed_items, bool> >; CGAL::SNC_point_locator_by_spatial_subdivision<SNC_decorator>::SNC_structure = CGAL::SNC_structure<CGAL::Simple_cartesian<double>, CGAL::SNC_indexed_items, bool>]’
[build] /root/Documents/project/geometricserver/function/intersectinfo/collidearea.cpp:94:1:   required from here
[build] /usr/local/include/CGAL/Nef_3/Bounding_box_3.h:85:16: error: ‘lo’ is not a member of ‘CGAL::Bounding_box_3<CGAL::Boolean_tag<true>, CGAL::Simple_cartesian<double> >::Base {aka CGAL::Box_intersection_d::Box_d<double, 3>}’
[build]        std::copy( q, q + 3, Base::lo );
[build]                 ^
[build] /usr/local/include/CGAL/Nef_3/Bounding_box_3.h:86:16: error: ‘hi’ is not a member of ‘CGAL::Bounding_box_3<CGAL::Boolean_tag<true>, CGAL::Simple_cartesian<double> >::Base {aka CGAL::Box_intersection_d::Box_d<double, 3>}’
[build]        std::copy( q, q + 3, Base::hi ); 
```

**Comment by sloriot** [comment]
Should be fixed by #4853

**Comment by haelue** [comment]
Thank you very much!
